src/resource_server/utils/common.py

```diff
@@ -63,3 +63,24 @@ def validate_schema(path_file: str) -> Dict[str, any]:
     validate(instance=instance,schema=schema)
 
     return instance
+
+
+def create_request_param_to_config(params: Dict) -> list:
+    parameters = list()
+    attributes = dict()
+
+    # Transform the values to the correct type 
+    for key in params.keys():
+        try:
+            attributes[key] = int(params[key])
+        except ValueError:
+            try: 
+                attributes[key] = float(params[key])
+            except ValueError:
+                # allow a str max of 8 characteres
+                attributes[key] = str(params[key])[:8]
+            
+        # Create the default structure for the parameters 
+        parameters.append({"name": key, "type": type(attributes[key]).__name__, "default": attributes[key]})
```

Comment:
This cloud be problematic as well....
e.g. if a parameter is a version number ... "1.10", the this parser would turn that into a float "1.0"